app/schemas/com.jerboa.db.AppDB/26.json

```diff
@@ -2,7 +2,7 @@
   "formatVersion": 1,
   "database": {
     "version": 26,
-    "identityHash": "bf20da38fc7d03e28121589dfb372eaf",
+    "identityHash": "98ab20c6cd84e7fedbb9eccc2e4c44b1",
```

Comment:
Do not change the existing scheme, up the version and push the new version.

I think it's somewhere in AppDB

Reply:
Can you explain this more please? Now I did this without changing the base, but I'm still curious what you meant.

Reply:
you changed the scheme before upping the version, (db classes). This caused it to generated a new scheme for 26. This scheme is needed to test the migrations. The test for example upgrades a database from 25 to 26 and then compares it to this json scheme format. Well If it contains changes from 27 it will no longer be accurate and the test will fail